config/session.php

```diff
@@ -155,7 +155,7 @@
     |
     */
 
-    'domain' => env('SESSION_DOMAIN', null),
+    'domain' =>  env('SESSION_DOMAIN', ''),
```

Comment:
sao lại sửa ntn?

Reply:
em sửa như vậy thì không bị lỗi 419 ạ

Reply:
```suggestion
    'domain' => env('SESSION_DOMAIN', ''),
```
sao ko sửa lại domain trong env???